src/components/DatePicker/DatePicker.tsx

```diff
@@ -384,4 +380,21 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
       }
     }
   }
+
+  @autobind
```

Comment:
I can't quite find the place where it's necessary to autobind this.

It won't make a massive difference but just not clear why it's autobind'd.